GistanTests/APIClient/GitHub/Response/GistFileTest.swift

```diff
@@ -35,22 +41,46 @@ class GistFileTest: XCTestCase {
             "raw_url": "https://gist.githubusercontent.com/hhyyg/9e18f2f16650d89aa37da10b06670d1e/raw/b57deff85fa2f56975a5eb4bf06a9ca2571e97b6/C%23%206%20exception%20filter.cs",
             "size": 917
         }
-    }
+    },
+    "owner": {
+      "login": "hhyyg",
+      "id": 8636660,
+      "avatar_url": "https://avatars0.githubusercontent.com/u/8636660?v=4",
+      "gravatar_id": "",
+      "url": "https://api.github.com/users/hhyyg",
+      "html_url": "https://github.com/hhyyg",
+      "followers_url": "https://api.github.com/users/hhyyg/followers",
+      "following_url": "https://api.github.com/users/hhyyg/following{/other_user}",
+      "gists_url": "https://api.github.com/users/hhyyg/gists{/gist_id}",
+      "starred_url": "https://api.github.com/users/hhyyg/starred{/owner}{/repo}",
+      "subscriptions_url": "https://api.github.com/users/hhyyg/subscriptions",
+      "organizations_url": "https://api.github.com/users/hhyyg/orgs",
+      "repos_url": "https://api.github.com/users/hhyyg/repos",
+      "events_url": "https://api.github.com/users/hhyyg/events{/privacy}",
+      "received_events_url": "https://api.github.com/users/hhyyg/received_events",
+      "type": "User",
+      "site_admin": false
+    },
+    "public" : true
 }
-"""
+""".data(using: .utf8)!
         
         //do
-        let result = try! JSONDecoder().decode(GistItem.self, from: convertToJSON(text: rawJson))
+        let result = try! decoder.decode(GistItem.self, from: rawJson)
```

Comment:
テストメソッドを`throws`にしてここは`try`にするのがセオリー。
(でも以前逆にそうすると結果がわかりにくくなったとかあったような気もする 🤔 )
[Avoiding force unwrapping in Swift unit tests — Swift by Sundell](https://www.swiftbysundell.com/posts/avoiding-force-unwrapping-in-swift-unit-tests)

Reply:
わからないので改めて確認。

`try!` にすると、画像のようにエラーの詳細がわかる：
![screen shot 2017-11-23 at 20 44 10](https://user-images.githubusercontent.com/8636660/33171624-4309834a-d090-11e7-8a2a-b7b04a24c8a8.png)

`try`にすると、次のようなメッセージがOutputにでて、エラーの原因の確認方法がわからず。：
```
<unknown>:0: error: -[GistanTests.GistFileTest testParseTest] : failed: caught error: The data couldn’t be read because it is missing.
```

今回の場合は前者にしました。